src/main/java/JingHao.java

```diff
@@ -0,0 +1,211 @@
+import commandexceptions.InvalidDeadlineCommandException;
+import commandexceptions.InvalidEventCommandException;
+import commandexceptions.InvalidTodoCommandException;
+import commandexceptions.JingHaoExceptions;
+
+import tasktype.Deadline;
+import tasktype.Event;
+import tasktype.Task;
+import tasktype.Todo;
+import tasktype.TaskList;
+
+import storage.Storage;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class JingHao {
+    private static final String LINE_SEP = "____________________________________________________________";
+
+    protected static TaskList taskList;
+    protected Scanner in;
+
+    public JingHao() {
+        try {
+            taskList = Storage.readFile();
+        } catch (IOException e) {
+            // Handle IOException here
+            System.out.println("Something went wrong: " + e.getMessage());
+            return;
+        } catch (JingHaoExceptions e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+        this.in = new Scanner(System.in);
+    }
+
+    public void start(){
```

Comment:
There should be a spacing between start() and the bracket. Same goes for other methods in this class and several other classes.